docker-cloud/cloud-swarm/register-swarms.md

```diff
@@ -0,0 +1,103 @@
+---
+description: how to register and unregister swarms in Docker Cloud
+keywords: register swarms, unregister swarms
+title: Register existing swarms
+---
+
+At this time, you cannot _create_ swarms from within Docker Cloud. However you can register existing swarms as part of the beta functionality.
```

Comment:
Change: "At this time, you cannot create swarms from within Docker Cloud. However you can register existing swarms as part of the beta functionality."

To: "With Beta Swarm mode, you can register existing swarms with Docker Cloud in order to easily manage a large number of Swarms running anywhere."